### ISSUE TITLE
Azure Manager for Monitor

### ISSUE THREAD

**Issue by asuresh4** [Azure/azure-libraries-for-java#78]
Hi,

QUESTION: I see that there's code for Monitor service in this repo here: https://github.com/Azure/azure-libraries-for-java/tree/master/azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor. But I don't find any documentation related to monitor here: https://azure.github.io/azure-sdk-for-java/overview-summary.html. Is it not supported?

Thanks!

**Comment by hovsepm** [comment]
Monitor service is available as Beta package in Java which does not fully implement Fluent interface pattern. It was mainly released to unblock customers who are currently blocked on the absence of Java SDK of any kind for Monitor Service. As a workaround for now you could check documentation for Monitor services in [other language SDKs](https://docs.microsoft.com/en-us/azure/#pivot=sdkstools&panel=sdkstools-all). They all are generated from the same source and should support the same functionality and expose same methods.

**Comment by hovsepm** [comment]
@jianghaolu could you provide steps on how to reference Stub Package in the project?

**Comment by anuchandy** [comment]
Jianghao is out of office this week.


The stub package can be used by adding following reference to POM:

```xml
<dependency>    
   <groupId>com.microsoft.azure</groupId>    
   <artifactId>azure-mgmt-monitor</artifactId>
    <version>1.4.0-beta</version>
</dependency>
```

Generated code can be consumed as:

```java
final File credFile = new File(System.getenv("AZURE_AUTH_LOCATION"));
ApplicationTokenCredentials credentials = ApplicationTokenCredentials.fromFile(credFile);
MonitorManager monitorManager = MonitorManager
    .configure()
    .withLogLevel(LogLevel.BODY_AND_HEADERS)
    .authenticate(credentials, credentials.defaultSubscriptionId());
```

```java
MonitorManagementClientImpl inner1 = monitorManager.inner();
MonitorClientImpl inner2 = monitorManager.innerEx();
```

**Comment by asuresh4** [comment]
Thank you @hovsepm and @anuchandy ! I'll have a look at this. However, would it also be possible for you to give an idea of when this would be available as part of the primary package? Thanks again guys!

**Comment by hovsepm** [comment]
ActivityLogs and Metris were shipped with v1.6. We are working on the other areas and they will be included in upcoming releases.
